spec/lib/unit_tests_utils/bosh_spec.rb

```diff
@@ -272,4 +284,26 @@ def bosh_release_output(release_name, release_version)
       end
     end
   end
+
+  describe ".execute_or_raise_error" do
+    context "when the command returns a zero status" do
+      it "does not raise a BOSH exception" do
+        expect(UnitTestsUtils::Bosh).to receive(:execute_or_raise_error).once.
```

Comment:
Hmm, am I wrong or are you just testing nothing here?

The class function `UnitTestsUtils::Bosh.execute_or_raise_error` should be tested and not be replaced by rspec magic.

Just provoke an error with a simple shell command:
```ruby
UnitTestsUtils::Bosh.execute_or_raise_error("ls /none-existing-dir", "the-msg")
# stdout, stderr, exit_status = Open3.capture3("ls /none-existing-dir")
# => ["", "ls: /none-existing-dir: No such file or directory\n", #<Process::Status: pid 78821 exit 1>]
```

And then have a successful run:
```ruby
UnitTestsUtils::Bosh.execute_or_raise_error("echo works", "works")
# irb(main):002:0>  stdout, stderr, exit_status = Open3.capture3("echo works")
# => ["works\n", "", #<Process::Status: pid 78812 exit 0>]
```

Reply:
You can also use `/usr/bin/env true` for a successful command and `/usr/bin/env false` for a failing one.